# Molecular: Phenotyping {#phenotyping}

In this lab "phenotyping"" generally refers to the various forms of gene expression analysis that we do.  What this means for molecular work is usually some form of RNA isolation and quantification.  Microbiome projects, for example, utilize 16S rRNA genes (DNA) to identify and quantify various microbial communities.  Behavioral phenotyping is usually gathered in an adjacent lab, and is analyzed alongside phenotyping/genotyping data that is associated with the same samples or specimen.

## Quantitative/Real-Time PCR

(**Note**:  _For our lab we use Roche's Universal Proble Library (preferred), and SYBR Green.  The qPCR protocol is written for the UPL system._)

Quantitative PCR (qPCR) or Real Time PCR (rtPCR) is an older technique that has been largely replaced with _Next Generation Sequencing (NGS)_ also known as _High Throughput Sequencing (HTS)_ for some applications.  Nevertheless, qPCR is still widely used for a multitude of  experimental procedures including validation of NGS/HTS results.  qPCR machines work the same as traditional thermocyclers, except they contain photodetectors for measuring fluorescent signals.  During the DNA amplification process the fluorescence signal increases indicating an increase in DNA concentration.  Using special software the qPCR machine can accurately calculate the starting concentration of the target gene.  Check out this [video](https://www.youtube.com/watch?v=YhXj5Yy4ksQ) to get a better understanding of qPCR.  Check [here](https://www.youtube.com/watch?v=DH7o9Df5_50) for a more detailed explanation.

To actually start a qPCR experiment for measuring RNA expression, several things need to take place:

1. [Tissue collection](#sample_collection)
2. [RNA has to be isolated](#rna_isolation).
3. cDNA libraries must be created from RNA samples.
4. Primers and Probes have to be designed, ordered, resuspended, and diluted to the proper concentrations. 
5. The experiment has to be [designed, and the workspace has to be prepared](#molecular_best_practices).
6. A program has to be created on the [LightCycler 96](#).

## RNA Sequencing

RNA sequencing is the process of identifying RNA expression in a given tissue of an organism.  An experiment utilizing RNA sequencing normally involves mutliple specimen and multiple tissues.  A more elaborate approach would be to also include RNA expression over time, which would of course require the specimen to be alive and would for obvious reasons be much more difficult and expensive.  Regardless, RNAseq can give valuable insights into the changes in RNA expression levels that might indicate disease or behavioral change.  To learn more see [Illumina's website](https://www.illumina.com/techniques/sequencing/rna-sequencing.html) or [RNA-Seq on Wikipedia](https://en.wikipedia.org/wiki/RNA-Seq).

For our purposes, RNA sequencing is normally carried out by the Genomics Core.  So the only required molecular work is RNA isolation.

## Single-Cell RNA Sequencing

More to come...

## 16S rRNA Gene (DNA) Sequencing

The 16S gene is used to transcribe a small part of the ribosomal RNA found in all organisms.  However, it's primary function in biological research is that of a biomarker for identifying microbiota.  After amplifying this gene in a sample of feces for example, different microbes can be identified through 16S rRNA gene sequencing and categorized as an OTU group, a species, or as a gene using specialized bioinformatics software.  Dr. Dan Knights from the University of Minnesota has a comprehensive youtube playlist dedicated to [Microbiome Discovery](https://www.youtube.com/playlist?list=PLOPiWVjg6aTzsA53N19YqJQeZpSCH9QPc) that you should watch if you would like to know more.

For our purposes, 16S rRNA sequencing is normally carried out by the Genomics Core.  So the only required molecular work is DNA isolation.
